docs/platforms/android/integrations/apollo/index.mdx

```diff
@@ -16,6 +16,12 @@ Capturing transactions requires that you first <PlatformLink to="/tracing/">set
 
 Sentry Apollo integration provides the `SentryApolloInterceptor`, which creates a span for each outgoing HTTP request executed with an [Apollo Android](https://github.com/apollographql/apollo-android) GraphQL client.
 
+<Note>
+
+This integration does not yet support <Link to="https://github.com/apollographql/apollo-kotlin/releases/tag/v4.0.0">Apollo4</Link>. If you're using Apollo4 and are interested in an integration for it, please track and upvote the <Link to="https://github.com/getsentry/sentry-java/issues/3662">related issue</Link>.
```

Comment:
Should we rephrase this to say it's only intended for Apollo v2? Same below for v3. My guess is, we'll have to build another module for Apollo v4.

Reply:
@adinauer updated wording, ptal 🙏